app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -901,7 +912,7 @@ def rbac_get_info
       @right_cell_text = _("%{model} \"%{name}\"") % {:model => ui_lookup(:model => "User"), :name => User.find(from_cid(id)).name}
       rbac_user_get_details(id)
     when "g"
-      @right_cell_text = _("%{model} \"%{name}\"") % {:model => ui_lookup(:model => "MiqGroup"), :name => MiqGroup.find(from_cid(id)).description}
+      @right_cell_text = _("%{model} \"%{name}\"") % {:model => ui_lookup(:model => "MiqGroup"), :name => MiqGroup.find(cid?(id) ? from_cid(id) : id).description}
```

Comment:
@lgalis i think from_cid method returns id as is in case it is not a compressed it, i don't think we need cid? check here